apps/auth_content/views.py

```diff
@@ -6,21 +6,27 @@
 from apps.common.utils import closedpod_user_check
 
 from .forms import ClosedPODContactInformationForm
-from .models import ClosedPODContactInformation
+from .models import ClosedPODContactInformation, ClosedPODHomePage
 
 
 @require_http_methods(["GET"])
 @user_passes_test(closedpod_user_check)
 def closedpod_contact_information(request):
     """Return the request.user's ClosedPODContactInformation."""
+    home_page = ClosedPODHomePage.objects.live().first()
+
     if hasattr(request.user, "closedpodcontactinformation"):
         contact_info = request.user.closedpodcontactinformation
     else:
         contact_info = ClosedPODContactInformation(user=request.user)
     return render(
         request,
         "auth_content/closedpod_contact_information.html",
-        {"contact_info": contact_info},
+        {
+            "contact_info": contact_info,
+            "show_closedpod_sidebar": True,
+            "closedpod_children_pages": home_page.get_children() if home_page else [],
```

Comment:
Could we test this for the following cases:
 - there is no `ClosedPODHomePage`
 - there is a `ClosedPODHomePage`, but it's not live
 - there is a live `ClosedPODHomePage` with no children
 - there is a live `ClosedPODHomePage` with children

 ?

Reply:
will do. working on it

Reply:
If a ClosedPODHomePage has not been created or is not live, do you think this page should error if navigated too. `404` or `400`?

Reply:
Good question. I don't know. I could see an argument for either one; I guess feel free to do whichever makes more sense to you

Reply:
Ok, that sounds good.